examples/global-tables/main.tf

```diff
@@ -41,11 +41,12 @@ resource "aws_kms_key" "secondary" {
 module "dynamodb_table" {
   source = "../../"
 
-  name             = "my-table-${random_pet.this.id}"
-  hash_key         = "id"
-  range_key        = "title"
-  stream_enabled   = true
-  stream_view_type = "NEW_AND_OLD_IMAGES"
+  name                                  = "my-table-${random_pet.this.id}"
+  hash_key                              = "id"
+  ignore_changes_global_secondary_index = true
```

Comment:
This example doesn't set `autoscaling_enabled` argument. Let's copy this module call and set `autoscaling_enabled = true` to see that the introduced feature works as expected.

Reply:
Good point, I'll fix that and get it tested.

Reply:
I've moved this over to the autoscaling example since it has everything we need.

Reply:
Please revert changes in this example, too.

Reply:
Done and done @antonbabenko